profiling/src/php_ffi.c

```diff
@@ -105,7 +104,53 @@ void ddog_php_prof_zend_mm_set_custom_handlers(zend_mm_heap *heap,
 #endif
 }
 
-zend_execute_data* ddog_php_prof_get_current_execute_data()
-{
+zend_execute_data* ddog_php_prof_get_current_execute_data() {
     return EG(current_execute_data);
 }
+
+#if PHP_VERSION_ID >= 80000
+static int ddog_php_prof_run_time_cache_handle = -1;
+#endif
+
+void ddog_php_prof_function_run_time_cache_init(const char *module_name) {
```

Comment:
I don't really understand if we had an alternative to having a C layer part. I'll come back to it.

Reply:
In this case, the C macros aren't available to Rust code through bindgen, so I made a little wrapper to expose it. Additionally, Rust doesn't like global mutable state like `ddog_php_prof_run_time_cache_handle`, but in this case it's simple enough and we already have C code for the previous point.

Reply:
Ah yes, you mentioned that! Thanks